client/lib/akismet/is-akismet-checkout.ts

```diff
@@ -0,0 +1,8 @@
+/*
+    The function isAkismetCheckout() is used to determine if the current page is a Akismet checkout page. 
+    It always returns false on the server side as window object is not available there (assumption that checkout pages are not rendered server-side).
+ */
+const isAkismetCheckout = () =>
```

Comment:
I'm surprised this didn't exist already, could definitely be used in several places right now 😅

Reply:
Yeah, I see that you found a new usage already! It's a simple but very handy check, I agree